blocks/blog-footer/blog-footer.js

```diff
@@ -0,0 +1,143 @@
+                    <i class="icons icon-wrapper">
+                        <svg xmlns="http://www.w3.org/2000/svg" data-name="Layer 1" viewBox="0 0 70 70"><path d="M28 22.61a3.67 3.67 0 01-3.66 3.66 3.62 3.62 0 01-3.61-3.66 3.67 3.67 0 013.61-3.72A3.72 3.72 0 0128 22.61zM21.37 49V28.45h5.95V49h-5.95zm14.76-20.55v1a8.78 8.78 0 017.64.21 8.17 8.17 0 014 6.85V49h-5.85V36.47a2.19 2.19 0 00-1-1.7c-2.34-1.33-4.78 1.06-4.78 1.06V49h-5.95V28.45h5.95z"></path></svg>
+                    </i>
+                </a>
+            </li>
+            <li class="share-item">
+                <a href="mailto:?subject=How to master delivering content at scale&amp;body=${window.location.href}" class="share-link share-mail" target="_blank" rel="noopener noreferrer">
+                    <i class="icons icon-wrapper ">
+                        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 70"><path d="M49 23H21c-1.3 0-2.3 1-2.3 2.3v19.4c0 1.3 1 2.3 2.3 2.3h28c1.3 0 2.3-1 2.3-2.3V25.3c0-1.3-1-2.3-2.3-2.3zm-.5 4.7v14.2l-7.1-7.1 7.1-7.1zm-19.9 7.1l-7.1 7.1V27.7l7.1 7.1zM23 44.2l7.5-7.5 4.5 4.5 4.4-4.5 7.5 7.5H23zm12-6.9L23.5 25.8h23L35 37.3z"></path></svg>
+                    </i>
+                </a>
+            </li>
+        </ul>`;
+  bFooter.append(shareFooter);
+  const shareTopics = document.createElement('div');
+  shareTopics.setAttribute('class', 'blogfooter-base');
+  const shareHeaderSpan = document.createElement('span');
+  shareHeaderSpan.setAttribute('class', 'blogfooter-header');
+  shareHeaderSpan.innerText = 'TOPICS';
+  shareTopics.append(shareHeaderSpan);
+  const shareSpan = document.createElement('span');
+  shareSpan.setAttribute('class', 'blogfooter-topics');
+
+  // get the configured tags
+  const tags = document.head.querySelectorAll('meta[property="article:tag"]');
+  tags.forEach((tag, index) => {
+    const topic = document.createElement('a');
+    topic.setAttribute('class', 'blogfooter-topics-link');
+    let linkHref = '/insights/blog?tag_filter=';
+    linkHref += toClassName((tag.getAttribute('content')));
+    topic.setAttribute('href', linkHref);
+    topic.innerText = tag.getAttribute(('content'));
+
+    shareSpan.append(topic);
+    if ((index + 1) !== (tags.length)) {
+      shareSpan.append(', ');
+    }
+  });
+
+  shareTopics.append(shareSpan);
+  bFooter.append(shareTopics);
+  const hrFooter = document.createElement('hr');
+  hrFooter.setAttribute('class', 'line-base');
+  bFooter.append(hrFooter);
+  const blogfooterAuthor = document.createElement('div');
+  blogfooterAuthor.setAttribute('class', 'blogfooter-authorprofile');
+  const author = await getProfile();
+  blogfooterAuthor.innerHTML = `<div>
+                <div class="authorprofile-base">
+                    <div class="authorprofile-image">
+                        <div class="nc-image-base">
+                            <div class="nc-image-container" itemtype="http://schema.org/ImageObject">
```

Comment:
What value does having the `itemtype` schema, and the series of wrapping divs, provide?